scripts/ci/publish.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 function image_push() {
     echo ${DOCKERHUB_TOKEN} | docker login --username sonobuoybot --password-stdin
-    make containers push
+    make push
```

Comment:
Eventually (when windows+linux nodes supported) we can't build both on the same machine. So presumably we've `make container` and `make win-container` in different machines and loaded the images for this step.